pkg/cmd/image/list.go

```diff
@@ -128,6 +129,21 @@ type imagePrintable struct {
 
 func printImages(ctx context.Context, client *containerd.Client, imageList []images.Image, options *types.ImageListOptions) error {
 	w := options.Stdout
+	var ImageList []images.Image
+	if options.GOptions.Namespace != "k8s.io" || options.All {
```

Comment:
Needs a comment to clarify how the behavior differs for k8s.io

Reply:
Needs a test too

Reply:
Ok, I'll add instructions and tests to `--kube-hide-dupe` to the feature refinement